lib/tasks/serve.js

```diff
@@ -52,13 +52,16 @@ module.exports = Task.extend({
       expressServer: expressServer
     });
 
-    return Promise.all([
-        liveReloadServer.start(options),
-        expressServer.start(options)
-      ]).then(function() {
+    // note Promise.all will hang under certain cirumstances, aka
```

Comment:
why would that hang? I could imagine the all rejects, unless their is an upstream bug not fulfilling or rejecting.